modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -33,14 +33,112 @@ This {release} version of the software and programming language should not be us
+This version of {proglang} includes syntax changes that might require you to update existing programs.
+
+Key updates for {proglang} include support for the following features and changes:
+
+* Simple object literals of the form `+{a = foo(); b = bar()}+` no longer bind to field names locally. 
+This change enables you to write expressions like this:
++
+....
+func foo(a : Nat) { return {x = x} }.									
+....
++
+However, this new syntax breaks short-hand expressions such as:
++
+....
+`+{a = 1; b = a + 1}+`
+....
++
+Short-hand expressions like the one above must now be written differently for your program to compile.
+For example, you might modify the expression to use an auxiliary declaration like this:
++
+....
+let a = 1; {a = a; b = a + 1}
+....
++
+Alternatively, you could modify the program to use object syntax like this:
++
+....
+{public let a = 1; public let b = a + 1}
+....
+
+* Free-standing blocks are no longer allowed.
++
+With this release, blocks are only allowed as sub-expressions of control flow expressions such as `+if+`, `+loop+`, and `+case+`.
+In all other places, braces are always considered to start an object literal.																													
+To use blocks that are not part of control flow expressions, you can use `+do {<block>}+` expressions.
++
+In this release, free-standing blocks that are not part of a control flow expressions display a warning but continue to compile.
+However, you should update your programs to replace the deprecated syntax to ensure your program continues to compile after the compiler enforces the syntax change.
+
+* Actor declarations are asynchronous and can only be used in asynchronous contexts.
++
+The return type of an actor class, if specified, must be an `+async+` actor type.
+To support actor declaration, the top-level context of an interpreted program is an asynchronous context, allowing implicit and explicit `+await+` expressions.
++
+This change mostly affects interpreted programs and compiled programs with explicit actor class return types.
+
+* Strict checking of `+utf8+` strings for improved Candid compliance.
+
+* More liberal parsing of leb128-encoded numbers
+
+* New `+Random+` library added to `+motoko-base+`.
+
+* Candid includes the ability to extend function with optional parameters in a backward-compatible way.
+
+* Injecting a value into an option type (`+? <exp>+`) no longer requires heap allocation in most cases. This removes the memory-tax of using iterators.																													
 
 For information about breaking changes that were introduced in previous releases, see <<Breaking changes>>.
 
 For information about known issues that were introduced in previous releases, see <<Known issues and limitations>>.
 
+=== Highlights of what's new in 0.6.14
+
+The 0.6.14 release primarily includes internal enhancements and fixes and updates to the version of {proglang} and the {proglang} base library that are supported. 
```

Comment:
```suggestion
=== Highlights of what's new in 0.6.16

The 0.6.16 release primarily includes internal enhancements and fixes and updates to the version of {proglang} and the {proglang} base library that are supported. 
```

Reply:
we've been piggy-backing the releases on top of each other, so I think this really is the description for the 0.6.14 release, but I'll have to double-check.